packages/cli/templates/uiPlugin/package.json

```diff
@@ -9,64 +9,59 @@
     "serve": "vcd-ext serve",
     "ng:serve": "ng serve"
   },
-  "license": "BSD-3-Clause",
+  "license": "<%- answers.license %>",
   "dependencies": {
-    "@angular/animations": "^6.1.0",
-    "@angular/common": "^6.1.0",
-    "@angular/compiler": "^6.1.0",
-    "@angular/core": "^6.1.0",
-    "@angular/forms": "^6.1.0",
-    "@angular/http": "^6.1.0",
-    "@angular/platform-browser": "^6.1.0",
-    "@angular/platform-browser-dynamic": "^6.1.0",
-    "@angular/router": "^6.1.0",
-    "@clr/angular": "^0.13.11",
-    "@clr/icons": "^0.13.11",
-    "@clr/ui": "^0.13.11",
-    "@ngrx/store": "^6.0.1",
-    "@vcd/bindings": "^9.1.1",
-    "@vcd/sdk": "^0.12.0",
+    "@angular/animations": "9.1.12 ",
+    "@angular/cdk": "8.2.1",
+    "@angular/common": "9.1.12",
+    "@angular/compiler": "9.1.12",
+    "@angular/core": "9.1.12",
+    "@angular/forms": "9.1.12",
+    "@angular/http": "6.1.0",
+    "@angular/platform-browser": "9.1.12",
+    "@angular/platform-browser-dynamic": "9.1.12",
+    "@angular/router": "9.1.12",
+    "@clr/angular": "2.3.8",
+    "@clr/icons": "2.3.8",
+    "@clr/ui": "2.3.8",
```

Comment:
Should we bump this up to 3.1.3 as this is the current core ui version?

Reply:
Done